docs/v6_upgrade.md

```diff
@@ -11,105 +11,119 @@ Webpacker used to configure Webpack indirectly, which lead to a [complicated sec
 This means you have to configure integration with frameworks yourself, but webpack-merge helps with this. See this example for [Vue](https://github.com/rails/webpacker#other-frameworks) and scroll to the bottom for [more examples](#examples-of-v5-to-v6).
 
 ## How to upgrade to Webpacker v6 from v5
-
-1. Consider changing from the v5 default for `source_entry_path`.
-  ```yml
-    source_path: app/javascript
-    source_entry_path: packs
-  ```
-  consider changing to the v6 default:
-  ```yml
-    source_path: app/javascript
-    source_entry_path: /
-  ```
-  Then consider moving your `app/javascript/packs/*` (including `application.js`) to `app/javascript/` and updating the configuration file. 
+1. Ensure you have a clean working git branch. You will be overwriting all your files and reverting the changes that you don't want.
+
+1. Upgrade the Webpacker Ruby gem and the NPM package
+
+   Note: [Check the releases page to verify the latest version](https://github.com/rails/webpacker/releases), and make sure to install identical version numbers of webpacker gem and `@rails/webpacker` npm package. (Gems use a period and packages use a dot between the main version number and the beta version.)
+
+   Example going to a specific version:
+
+   ```ruby
+   # Gemfile
+   gem 'webpacker', '6.0.0.rc.7'
+   ```
+
+   ```bash
+   bundle install
+   ```
+
+   ```bash
+   yarn add @rails/webpacker@6.0.0-rc.7 --exact
+   ```
+
+   ```bash
+   bundle exec rails webpacker:install
+   ```
+
+1. There is now a single default configuration file of `config/webpack/webpack.config.js`. Previously, the config file was set
+  to `config/webpack/#{NODE_ENV}.js`. In the `config/webpack/` directory, you can either refactor your code in `test.js`, `development.js`, and `production.js` to a single file, `config.webpack.js` or you can replace the contents of `config/webpack/config.webpack.js` to conditionally load the old file based on the NODE_ENV with this snippet:
```

Comment:
i think `config.webpack.js` should be `webpack.config.js`

Reply:
I agree! Per the docs: https://webpack.js.org/configuration/, `webpack.config.js` is what's used.